test/functional-portable/corerp/cloud/mechanics/aws_mechanics_test.go

```diff
@@ -32,7 +32,7 @@ func Test_AWSRedeployWithUpdatedResourceUpdatesResource(t *testing.T) {
 	name := "radiusfunctionaltestbucket-" + uuid.New().String()
 	creationTimestamp := functional.GetCreationTimestamp()
 
-	test := shared.NewRPTest(t, name, []shared.TestStep{
+	test := corerp.NewRPTest(t, name, []corerp.TestStep{
```

Comment:
So does this mean that all of the other buckets of tests are going to call `corerp.NewRPTest`. Do we have a plan to change this?

Reply:
Was planning to change this later but I made the change in this PR now.